ctapipe/utils/datasets.py

```diff
@@ -142,12 +152,55 @@ def get_dataset_path(filename):
 
         return ctapipe_resources.get(filename)
 
+    # last, try downloading the data
+    try:
+        return download_file_cached(
+            filename,
+            default_url="http://cccta-dataserver.in2p3.fr/data/ctapipe-extra/v0.3.1/",
```

Comment:
Perhaps at least make this default url a module-level variable (e.g `DEFAULT_DOWNLOAD_URL`), so later we can change it more easily.  Or evem, anticipating selectable datasets, add `DATASET_VERSION = 'v0.3.1'` and then use 

```py3
default_url=f"http://cccta-dataserver.in2p3.fr/data/ctapipe-extra/{DATASET_VERSION}"`
```

That way later we can test other versions by just setting `ctapipe.utils.datasets.DATASET_VERSION` 

Another option would be to do what you do later with ENV_PREFIX, and add the ability to set envvars like `CTAPIPE_DATA_VERSION=v0.3.1` and/or `CTAPIPE_DATA_URL`

Reply:
CTAPIPE_DATA_URL is already builtin.